lib/cmds/utils.spec-unit.ts

```diff
@@ -38,10 +24,10 @@ describe('utils', () => {
     it('should create the default providers', () => {
       const argv = {
         _: ['foobar'],
-        chrome: true,
-        gecko: true,
-        standalone: true,
-        versions: {gecko: '0.16.0', chrome: '2.20'},
+        chromedriver: true,
```

Comment:
I like the change in naming here

Reply:
Me too. I was confused with chrome, chromedriver, chrome_logs and setting aliases makes sense without breaking functionality.